modules/op-results-concepts.adoc

```diff
@@ -0,0 +1,172 @@
+// This module is included in the following assembly:
+//
+// *cicd/pipelines/using-tekton-results-for-openshift-pipelines-observability.adoc
+
+:_content-type: CONCEPT
+[id="results-concepts_{context}"]
+= {tekton-results} concepts
+
+{tekton-results} archives pipeline runs and task runs in the form of results and records.
+
+For every `PipelineRun` and `TaskRun` custom resource (CR) that completes running, {tekton-results} creates a _record_.
+
+A _result_ can contain one or several records. A record is always a part of exactly one result.
+
+A result corresponds to a pipeline run, and includes the records for the `PipelineRun` CR itself and for all the `TaskRun` CRs that were started as a part of the pipeline run.
+
+If a task run was started directly, without the use of a pipeline run, a result is created for this task run. This result contains the record for the same task run.
+
+Each result has a name that includes the namespace in which the `PipelineRun` or `TaskRun` CR was created and the UUID of the CR.  The format for the result name is `<namespace-name>/results/<parent-run-uuid>`. In this format, `<parent-run-uuid>` is the UUUD of a pipeline run or else of a task run that was started directly.
```

Comment:
I believe "else" is needed, because otherwise in "of a pipeline run or a task run that was started directly" the clause "that was started directly" applies to both the pipeline run and the task run. In this event it applies only to the task run.